domains/tech4file.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "kubsai",
+           "email": "",
+           "discord": "477781395922485248",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.rjBMItaMqXDR5sgtxH1YnZZ9mYBhx3cbEQzddCZUiMNBR46LhAZ2AhH8wCbD6SG2zvXxFAs_37Lj-kdKpTwqvslDWBwu-USKw744-I9JSYOo5AOp07XOeMBub_hlhDzTs7c71UEp0Zn2aWcNT164tJZ9FtrrGz1cOlA1FrgOY6IiddK6Kpxc9WNfyKrFcvI36qFxp4Tb8S0Ibybm61KJbstlgbjxwB6KggPTHsywFOhw_Qefe9-QptaoPZRiruXHzvfWYoEzDVjlu7sfXbbyTfla9P6KMldZqhzfYLB1urG-iUnbE5lXWHpPUHS3Hg6-_zWzBIMn65gKfmXrp-uV7A.hW8GxTh4cATNDhwiGbFWyg.JoRpaJgGT1hqgC16L__tz931OK9opx5RmAWfsoe3QHUsiqCQM8xLkxUldZUX_UwYQWD0VZHxkk6KXYFfp2NwtQ.Q4d-6jP2DF2q_-6B8o-u5Q"
+        },
+    
+        "record": {
+            "A": ["185.27.134.230"]
```

Comment:
Page doesn't work, please give a preview of your website.

Reply:
i have created an A record from the list in discord and A record i can give only ip address where my website is hosted and i gave server ip address. so i can't give an exact website address to the A record.